src/gui/properties/propertieswidget.h

```diff
@@ -82,7 +82,7 @@ public slots:
     void readSettings();
     void saveSettings();
     void reloadPreferences();
-    void openDoubleClickedFile(const QModelIndex &index) const;
+    void openDoubleClickedItem(const QModelIndex &index) const;
```

Comment:
Has `DoubleClicked` part any meaning?

Reply:
Not really, what about rename it to `openItem()`?